regions/io/ds9/core.py

```diff
@@ -133,7 +133,7 @@ def _convert_sky_coords(self):
         frame = coordinates.frame_transform_graph.lookup_name(self.coordsys)
 
         lon, lat = zip(*parsed_angles)
-        if hasattr(lon, '__len__') and hasattr(lon, '__lat__') and len(lon) == 1 and len(lat == 1):
+        if hasattr(lon, '__len__') and hasattr(lat, '__len__') and len(lon) == 1 and len(lat) == 1:
```

Comment:
wow, how did this original ever get through testing? Could you try to find a test to add that will cover this change?

Reply:
ya..i too was thinking the same. I was not expecting such a trivial error. Almost took me two hours to get to this manually .! 
Well, how do you debug generally?

Reply:
Most of the tests that i came across asserted with the value of the radius.
Only when i wrote the tests for the previous issue , i came across this bug.
I haved changed my tests and i think is sufficient to cover this change.